### PR TITLE
Adding splash window with "new project", "open project" and "recent project"

### PR DESCRIPTION
**Pull request by lormniczek**
Create window with "new project", "open project", "recent project"

### CODE REVIEW

File: violetproduct-swing/src/main/java/com/horstmann/violet/application/gui/SplashFrame.java [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * This desktop frame contains panes that show menu to create/open projects.
+ *
+ * @author Maciej Kupniewski
```
Comment (abizukojc) [inline violetproduct-swing/src/main/java/com/horstmann/violet/application/gui/SplashFrame.java]: This should be removed

Comment (abizukojc) [inline violetproduct-swing/src/main/java/com/horstmann/violet/application/gui/SplashFrame.java]: This should be removed

Comment (abizukojc) [inline violetproduct-swing/src/main/java/com/horstmann/violet/application/gui/SplashFrame.java:32]: This should be removed

**Review by abizukojc — COMMENTED** [review]
Check the comment

File: violetproduct-swing/src/main/java/com/horstmann/violet/application/gui/SplashFrame.java (hunk 1 above)
Comment (RoughTomato) [reply violetproduct-swing/src/main/java/com/horstmann/violet/application/gui/SplashFrame.java:32]: I second that

**Review by orchowski — APPROVED** [review]

**Pull request merged**